package.json

```diff
@@ -86,13 +84,20 @@
     "ajv": "^4.0.0",
     "babel-polyfill": "^6.9.1",
     "bluebird": "^3.4.6",
+    "csv": "^1.1.0",
+    "csv-stringify": "^1.0.4",
     "debug": "^2.2.0",
+    "exceljs": "^0.2.35",
+    "extract-zip": "^1.5.0",
     "ghooks": "^1.0.3",
+    "glob": "^7.0.5",
+    "iconv-lite": "^0.4.13",
     "jszip": "^3.0.0",
-    "lodash": "^4.6.1",
+    "lodash": "^4.16.6",
```

Comment:
Underscore is not used anymore - I removed `underscore` from package.json, reinstalled dependencies and tried to run tests and it works without it.